src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/GoogleServicesDAO.scala

```diff
@@ -39,7 +38,5 @@ trait GoogleServicesDAO extends ReportsSubsystemStatus {
 
   def publishMessages(fullyQualifiedTopic: String, messages: Seq[String]): Future[Unit]
 
-  def getBucket(bucketName: String, petKey: String): Option[Bucket]
```

Comment:
I can just leave this + its implementations if that is better. It is not used anywhere else in orch which is why I deleted it.